frameworks/PHP/reactphp/benchmark_config.json

```diff
@@ -20,6 +20,25 @@
       "display_name": "reactphp",
       "notes": "",
       "versus": "php"
+    },
+    "libuv": {
```

Comment:
Is `default` required or can it be renamed `libev` to have consistent naming and clarify wich event loop extension is used in each test?

Reply:
Do you prefer libuv like default ??
It's OK , we can change it !!

But for me it's not the important think !!

Reply:
Like you want !!

Reply:
But firs check the results !!!

Reply:
Default is so  !! default
If you want to change it, make it !!
But I think it's not a good option.

Happy new year !!!

Reply:
It's not possible, default and the rest of variants !!

Reply:
Wait to check the results, and later change it !!

Reply:
If anybody want more information, and I think that's correct.
Then they need to check the repo, than all is clear  !!!

Reply:
> Default is so !! default If you want to change it, make it !! But I think it's not a good option.

Was just wondering if it was just a label or more.

> Happy new year !!!

Happy new year!



> Do you prefer libuv like default ?? It's OK , we can change it !!
> 
> But for me it's not the important thing !!

Nah ev is fine as default, and really liking how you make it clear wich one is wich :+1: .